docs/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.rst

```diff
@@ -244,15 +244,15 @@ by ``DFAlgNode`` based trees. Scalar computations are described by ``BaseExpr``
 * :doc:`Frame nodes <df_algebra>`
 * :doc:`Expression nodes <expr>`
 
-Interactions with OmniSci engine are done using ``OmnisciServer`` class. Queries use serialized
+Interactions with OmniSci engine are done using ``OmnisciWorker`` class. Queries use serialized
```

Comment:
Do we want to rename the `OmnisciOnNative` to `PyhdkOnNative` execution or something like that?